src/app/cart/cart.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { CartService } from '../Services/cart.service';
+
+@Component({
+  selector: 'app-cart',
+  templateUrl: './cart.component.html',
+  styleUrls: ['./cart.component.scss']
+})
+export class CartComponent implements OnInit {
+
+  groceryCart = [];
+  totalPrice: number;
+
+  constructor(
+    private _cartService: CartService,
+    private router: Router
+  ) { }
+
+  ngOnInit() {
+    this._cartService.itemCount$.subscribe(data => {
+      this.groceryCart = data;
+    });
+  }
+
+  removeProduct(product) {
+    const productPosition = this.groceryCart.indexOf(product);
+    if (productPosition != -1) {
```

Comment:
This operation should happen within the cart service

Reply:
done